### PR TITLE
Feature/elk

### PR DESCRIPTION
**Pull request by AyadiAmen**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [X] Variables are documented in the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,4 +66,5 @@ Each requirement is configured with the options provided by that Chart. Please c
 | `pgadmin.enabled`                                                           | Enable PgAdmin                                                                                                     | `true`                          |
 | `openldap.enabled`                                                          | Enable openldap                                                                                                    | `true`                          |
 | `phpldapadmin.enabled`                                                      | Enable phpLDAPadmin                                                                                                | `true`                          |
+| `ElasticStack.enabled`                                                      | Enable Elastic-stack                                                                                                | `true`                          |
```
Comment (alexnuttinck) [inline README.md:69]: @AyadiAmen Could you change `ElasticStack` to `elasticstack` ? I think it is better to use lowercases.

File: requirements.yaml [hunk 2]
```diff
@@ -40,3 +40,7 @@ dependencies:
   version: ~0.1.2
   repository: https://cetic.github.io/helm-charts/
   condition: phpldapadmin.enabled
+- name: elastic-stack
+  version: ~1.8.0
+  repository: https://kubernetes-charts.storage.googleapis.com/
+  condition: ElasticStack.enabled
```
Comment (alexnuttinck) [inline requirements.yaml]: @AyadiAmen the same here.

**Comment by AyadiAmen** [comment]
@alexnuttinck it's done

**Comment by alexnuttinck** [comment]
Thanks @AyadiAmen!

**Pull request merged**